netbout-web/pom.xml

```diff
@@ -744,24 +744,15 @@
                                 <exclude>xml:/src/main/resources/lang/.*</exclude>
                                 <exclude>duplicatefinder:.*</exclude>
                                 <!--
-                                @todo #961:30min These excludes were added
-                                 during upgrade to qulice 0.15.2. New version of
-                                 qulice adds new violations that need to be fixed.
-                                 There are also excludes added to netbout-client/pom.xml
-                                 (lines 192 - 207) related to this puzzle.
-                                 They also need to be fixed in order to solve this puzzle.
+                                @todo #1012:30min These excludes were added because
+                                 of the upgrade to qulice 0.16.2. The new version of qulice
```

Comment:
@mkordas There is a commit which updates netbout/pom.xml to qulice 0.16.2. Actually all violations with qulice 0.15.2 (the ticket is addressed to) are fixed (exception some NonStaticMethod). The puzzle is only for violations which are related to 0.16.2